e2e/internal/e2e/singularitycmd.go

```diff
@@ -0,0 +1,355 @@
+// Copyright (c) 2019, Sylabs Inc. All rights reserved.
```

Comment:
If this is how you plan to run singularity you need to add traceability like what is included in `internal/pkg/test/exec/exec.go`. That would be a preferred way to execute over stdlib so we can log all commands run.

Reply:
Done